gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -57,6 +61,16 @@
     this.featureSheetState = new MutableLiveData<>();
     this.activeProject = LiveDataReactiveStreams.fromPublisher(dataRepository.getActiveProject());
     this.navigator = navigator;
+    this.addFeatureClicks = PublishSubject.create();
+
+    disposeOnClear(
+        addFeatureClicks
+            .switchMapSingle(Result.wrapErrors(dataRepository::addFeature))
+            .subscribe(Result.unwrapErrors(this::showFeatureSheet, this::onAddFeatureError)));
+  }
+
+  private void onAddFeatureError(Throwable throwable) {
+    Log.d(TAG, "Couldn't add feature.");
```

Comment:
Please add TODO to show something to the user if this happens, and please also `Log.e(TAG, "...", throwable)`;

Reply:
Thanks, done!